infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/endpoints/AsyncApiResponse.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.infrastructure.restapi.endpoints;
+
+public class AsyncApiResponse {
+  final int responseCode;
+  final String responseBody;
```

Comment:
This should be an `Object` not a `String` as the `RestApiRequest` should handle serialisation with the appropriate type based on the status code (same as it does for `respondOk` now).